main.go

```diff
@@ -160,14 +161,41 @@ func main() {
 		fmt.Printf("Current task description: \n%+v \n", taskDesc)
 	}
 
-	containerDef := taskDesc.TaskDefinition.ContainerDefinitions[0]
-	oldImage := containerDef.Image
-	{
-		x := *targetImage
-		if *targetImage == "" {
-			x = fmt.Sprintf("%s:%s", *repoName, *sha)
+	var containerDef *ecs.ContainerDefinition
+	var oldImage *string
+	var i int
+	// multiContainer service
+	if *multiContainer {
+		containerDef = taskDesc.TaskDefinition.ContainerDefinitions[0]
+		oldImage = containerDef.Image
+		{
```

Comment:
Why the extra block? It allows `x` to have a tighter scope, but it would only be scoped inside the `if` anyway, which isn't that much more.

Reply:
the extra bloc was already in the original code.. I guess I blindly copied and pasted